drivers/SmartThings/zigbee-switch/src/aqara-light/init.lua

```diff
@@ -11,7 +11,8 @@ local PRIVATE_ATTRIBUTE_ID = 0x0009
 local MFG_CODE = 0x115F
 
 local FINGERPRINTS = {
-  { mfr = "LUMI", model = "lumi.light.acn004" }
+  { mfr = "LUMI", model = "lumi.light.acn004" },
+  { mfr = "Aqara", model = "lumi.light.acn014" }
```

Comment:
Is this supposed to be Lumi or is this unique to this device because it is a CN device?

Reply:
The manufacturer of this device is Aqara.It is a CN device.

Reply:
Hi, @Hongming6 @lelandblue @greens 

When I test light device, i noticed there is a dimmer rebound issue.
I'm not sure if this issue also exist on `{ mfr = "LUMI", model = "lumi.light.acn004" }`
which passed WWST early.

If EdgeDriver send one `MoveToLevelWithOnOff` command to Aqara's bulb,
it responses multiple times with different value which cause the dimmer rebound issue.

One possible fixup is setting `transition_time`.
```
local function set_level_handler(driver, device, cmd)
  local level = math.floor(cmd.args.level / 100.0 * 254)
  local dimming_rate = 0x0000

  device:send(Level.commands.MoveToLevelWithOnOff(device, level, dimming_rate))
end
local aqara_light_handler = {
  NAME = "Aqara Light Handler",
  lifecycle_handlers = {
    added = device_added,
    doConfigure = do_configure
  },
  capability_handlers = {
    [capabilities.switchLevel.ID] = {
      [capabilities.switchLevel.commands.setLevel.NAME] = set_level_handler
        }
  },
  can_handle = is_aqara_products
}

```

Reply:
In general we have found this behavior of reporting intermediate states during transition acceptable as long as the final value is correct.

Reply:
Hi, @greens （CC： @liuyd96  @Hongming6  ）
Is it possible for us to modify it this way 

> Hi, @Hongming6 (CC: @lelandblue @greens )
> 
> When I test Aqara's light, i noticed there is a dimmer rebound issue. I'm not sure if this issue also exist on `{ mfr = "LUMI", model = "lumi.light.acn004" }` which passed WWST early or this issue should block WWST.
> 
> If EdgeDriver send one `MoveToLevelWithOnOff` command to Aqara's bulb, Aqara's bulb responses multiple times with different value which causes the dimmer rebound issue.
> 
> One possible fixup is setting `transition_time`.
> 
> ```
> local function set_level_handler(driver, device, cmd)
>   local level = math.floor(cmd.args.level / 100.0 * 254)
>   local dimming_rate = 0x0000
> 
>   device:send(Level.commands.MoveToLevelWithOnOff(device, level, dimming_rate))
> end
> local aqara_light_handler = {
>   NAME = "Aqara Light Handler",
>   lifecycle_handlers = {
>     added = device_added,
>     doConfigure = do_configure
>   },
>   capability_handlers = {
>     [capabilities.switchLevel.ID] = {
>       [capabilities.switchLevel.commands.setLevel.NAME] = set_level_handler
>         }
>   },
>   can_handle = is_aqara_products
> }
> ```

Reply:
If you want to include that in this sub-driver that is fine.

Reply:
Ok, it has been modified